src/cheri/introduction.adoc

```diff
@@ -30,115 +30,127 @@ be brute forced.
-endif::support_varxlen[]
-<<section_cheri_priv_crg_ext,{cheri_priv_crg_ext}>>^1^:: CHERI extension for capability revocation on RISC-V harts supporting page-based virtual-memory (implies {cheri_priv_vmem_ext}).
-
-^1^ {cheri_priv_crg_load_tag_ext} is available for improved software revocation performance if {cheri_priv_crg_ext} is implemented.
-
-==== Debug specifications (#need to move to a separate document#)
+{cheri_base_ext_name} is defined as the base ISA that all CHERI RISC-V implementations must support.
+{cheri_default_ext_name} and {cheri_priv_crg_ext} are examples of optional extensions in addition to
+{cheri_base_ext_name}.
 
-<<section_debug_integration_ext,Sdext ({cheri_base_ext_name})>>:: Debug specification for integrating CHERI with external debug mode (implies, and required by, Sdext).
-<<section_debug_integration_trig,{cheri_priv_debug_trig}>>:: Debug specification for integrating CHERI with debug triggers (implies, and required by, Sdtrig).
+In some cases adding an existing extension, such as H, causes additional instructions to be added.
 
-CAUTION: The extension names are provisional and subject to change.
+In other cases adding an existing extension, such as C, causes additional instructions to be added and existing instructions to have modified behavior.
 
-==== Extension and specification status
+==== Stable Extensions and Specifications
 
-.Unprivileged {cheri_base_ext_name} extension and specification status and summary
+.Unprivileged stable {cheri_base_ext_name} extensions
 [#unpriv-extension-status,reftext="Extension Status and Summary"]
-[options=header,align=center,width="90%",cols="25,15,52"]
+[options=header,align=center,width="90%"]
 |=============================================================================================================================================================
-| Extension                                                 | Status        | Comment
-|<<rv32y,{cheri_base_ext_name}>>                            | Stable        | This extension is a candidate for freezing
-|<<section_rvy_sentry_insn_ext>>                            | Stable        | This extension is a candidate for freezing
-|<<section_cheri_hybrid_ext,{cheri_default_ext_name}>>      | Stable        | This extension is a candidate for freezing
-|<<section_zyseal>>                                         | Experimental  | This extension is not yet a candidate for freezing
-|<<section_xycheriot>>                                      | Experimental  | This extension is not yet a candidate for freezing
+| Extension or Specification                                | Description
+|<<rv32y,{cheri_base_ext_name}>>                            | Base ISA
+|<<app_cap_description,{rvy_default_encoding_name}>>        | Default capability encoding
+|<<section_rvy_sentry_insn_ext>>                            | Sentry extension for CFI
```

Comment:
Not just CFI...

Reply:
also true - please suggest a suitable description